### PR TITLE
feat: add create-slug for all production apps, change deploy target b…

### PR DESCRIPTION
**Pull request by abtris**
# DON'T MERGE 

## just review need it

…y new order

- https://trello.com/c/CdJ26idX/821-rename-heroku-apps-accoridng-to-envs-on-opc-side

### CODE REVIEW

File: blackbelt/deployment.py [hunk 1]
```diff
@@ -9,11 +9,11 @@ def deploy_staging():
 
     post_message("Deploying branch %s to staging" % branch_name, "#deploy-queue")
 
-    check_call(['grunt', 'deploy', '--app=apiary-staging-pool1', '--force', "--branch=%s" % branch_name])
```
Comment (Almad) [inline blackbelt/deployment.py]: @abtris I thought `-pool1` is now used for "testing", is that true or not?

Comment (abtris) [reply blackbelt/deployment.py]: No, after rename `-pool1` will be plain staging.

File: blackbelt/deployment.py [hunk 2]
```diff
@@ -9,11 +9,11 @@ def deploy_staging():
...
+    check_call(['grunt', 'deploy', '--app=apiary-staging', '--force', "--branch=%s" % branch_name])
 
 def deploy_production():
     post_message("Deploying to production", "#deploy-queue")
 
-    check_call(['grunt', 'deploy'])
-    check_call(['grunt', 'deploy', '--app=apiary-staging'])
+    check_call(['grunt', 'deploy'])
```
Comment (miiila) [inline blackbelt/deployment.py]: NIT: Empty tab

**Comment by miiila** [comment]
https://github.com/apiaryio/apiary/pull/5129#issuecomment-347176903 is valid here as well.

**Pull request merged**